apps/src/maze/mazeMap.js

```diff
@@ -24,16 +24,23 @@ MazeMap.prototype.forEachCell = function (cb) {
   });
 };
 
+MazeMap.prototype.getCell = function (x, y) {
+  return this.currentStaticGrid[x] && this.currentStaticGrid[x][y];
```

Comment:
I could very well be doing something wrong, but I get an exception here if I delete 'Value' but forget to clear 'Range':

<img width="236" alt="screen shot 2016-07-06 at 10 51 13 pm" src="https://cloud.githubusercontent.com/assets/413693/16643726/2e42c676-43cc-11e6-83b6-68e09ddcd7f7.png">
`Cannot read property '0' of undefined`